index.html

```diff
@@ -4,7 +4,46 @@
 	<title>GitHub Pages Site</title>
 </head>
 <body>
-<h1>Under Construction</h1>
+<nav> 
+	<a href="index.html">Home</a>
+	<a href="additional.html">Additional</a>
+</nav>
+
+<div>
+	<h1>AYA-PHANT</h1>
+	<p>Aya The Elephant</p>
+</div>
+
+<img src="images/ayaphant.jpg" alt="aya in elephant costume"/>
```

Comment:
This `img` should be inside the div above probably, something other then being in the body tag.